CHANGELOG.md

```diff
@@ -1308,7 +1308,7 @@ To see unreleased changes, please see the [CHANGELOG on the main branch guide](h
 ### Added
 
 - `module` argument to `pyclass` macro. [#499](https://github.com/PyO3/pyo3/pull/499)
-- `py_run!` macro [#512](https://github.com/PyO3/pyo3/pull/512)
+- `py_run_bound!` macro [#512](https://github.com/PyO3/pyo3/pull/512)
```

Comment:
This needs reverting (but I'll wait to make any changes here until your first pass of review is finished).

Reply:
(Just removed this during rebase.)